speechbrain/lobes/models/transformer/Transformer.py

```diff
@@ -541,6 +542,8 @@ def forward(
         src_key_padding_mask : tensor
             The mask for the src keys per batch (optional).
         """
+        assert dct_config is None, "DCT unsupported for this encoder"
```

Comment:
But wait, then we have it defined in TransformerASR?

Reply:
The confusing part is that `TransformerEncoder` is specifically "the" classic transformer encoder class, not an interface like `TransformerInterface` is.

`ConformerEncoder` (and the others) have no relationship with `TransformerEncoder`.